include/power_grid_model/math_solver/iterative_current_pf_solver.hpp

```diff
@@ -0,0 +1,182 @@
+// SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+// Check if the name means anything
+#pragma once
+#ifndef POWER_GRID_MODEL_MATH_SOLVER_ITERATIVE_CURRENT_PF_SOLVER_HPP
+#define POWER_GRID_MODEL_MATH_SOLVER_ITERATIVE_CURRENT_PF_SOLVER_HPP
+// To avoid unused parameter and variable error
+#define UNUSED(x) (void)(x)
+
+/*
+Iterative Power Flow
+I = YU
+Split this with U_1 having all source buses, U_2 having all other buses
```

Comment:
The symbol used in Werners paper is less intuitive.

Please just describe without split the matrix.